kernel/george.c

```diff
@@ -46,6 +46,7 @@ void initialize_idt (void) {
   register_isr(0x20, timer_isr, 0);
   register_isr(0x21, kbd_isr, 0);
   register_isr(0x28, rtc_isr, 0);
+  register_isr(0x2B, ac97_isr, 0);
```

Comment:
unlike the lower ISRs that are all commented these aren't.

Perhaps outside scope for this patch - but maybe this should be a struct/array with constants that aren't magic.